cloudwash/providers/aws.py

```diff
@@ -19,7 +24,10 @@ def cleanup(**kwargs):
         dry_data['VMS']['skip'] = []
         for items in data:
             dry_data[items]['delete'] = []
-        with compute_client("aws", aws_region=region) as aws_client:
+
+        # Differentiate between the cleanup region to the client region, if client_region defined
+        r = client_region or region
```

Comment:
The `client_region` which is being used initially to fetch all the region details should be used at line https://github.com/RedHatQE/cloudwash/pull/134/files#diff-2c95ff1a0a20bbed48adf7449090cf7008c6a53371753a650eece11a6ccd6dd0R20

Reply:
Resolved in: https://github.com/RedHatQE/cloudwash/pull/134/commits/8c9bbaec10253c678a5dea14d92bfa86f8293f38